donate.html

```diff
@@ -205,6 +205,8 @@ <h4 class="ak-donate-area-step-1">Donation (Optional)</h4>
                           £){% endfilter %}
                           {% elif currency == "USD" %} {% filter ak_text:"general_us_dollar" %}U.S. Dollar (USD
                           $){% endfilter %}
+                          {% elif currency == "JPY" %} {% filter ak_text:"general_ja_yen" %}Japanese Yen (JPY ¥){% endfilter %}

```

Comment:
would this also need corresponding translation strings added? https://act.350.org/admin/core/language/7/change/

Reply:
good question, will ask Rachel about this!